feature-group-definitions/picture-in-picture.yml

```
@@ -1,5 +1,5 @@
-name: Picture-in-picture
-description: The picture-in-picture API allow websites to create a floating, always-on-top video window.
+name: Video picture-in-picture
+description: The picture-in-picture API allow websites to create a floating, always-on-top video window. Also known as PiP.
```

```suggestion
description: The picture-in-picture API allow websites to create a floating, always-on-top video window. Also known as PiP or pop-out video.
```

Ran across this term while doing some searches for my previous comment. [Vivaldi user docs](https://help.vivaldi.com/desktop/media/pop-out-video/).